### PR TITLE
Use chrono::DateTime for secrets

### PR DESCRIPTION
**Pull request by A6GibKm**
I am not sure if adding another dependency is desired.

Its somewhat weird to return the number of seconds since the unix epoch in anything but as a date.

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
I am not a fan of depending on chrono for this very small bit; using Duration is fine and people can convert it to a date using whatever library they are using

**Comment by A6GibKm** [comment]
Ok then ill add the methods as duration

**Comment by A6GibKm** [comment]
Done.

File: src/portal/item.rs [hunk 1]
```diff
@@ -88,6 +88,18 @@ impl Item {
         self.secret = secret.as_ref().to_vec();
     }
 
+    /// The UNIX time when the item was created.
+    pub fn created(&self) -> Duration {
```
Comment (bilelmoussaoui) [inline src/portal/item.rs:92]: Can't we have a Duration directly in the struct?

Comment (A6GibKm) [reply src/portal/item.rs:92]: Those are serialized/deserialized, no idea tbh

Comment (bilelmoussaoui) [reply src/portal/item.rs:92]: Right, Duration is serialized as a `{"secs": 10, "nanos": 20}` but implementing a custom Serialize/Deserialize for Item should be pretty easy

Comment (bilelmoussaoui) [reply src/portal/item.rs:92]: Let us land this as is for now

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request merged**